docs/index.md

```diff
@@ -0,0 +1,7 @@
+# Django Commons
+
+## Repositories
+
+- [django-commons/controls](https://github.com/django-commons/controls)
+- [django-commons/membership](https://github.com/django-commons/membership)
+- [django-commons/best-practices](https://github.com/django-commons/best-practices)
```

Comment:
```suggestion
- [django-commons/membership](https://github.com/django-commons/membership)
- [django-commons/best-practices](https://github.com/django-commons/best-practices)
- [django-commons/controls](https://github.com/django-commons/controls)
```
Since we don't have them alphabetical, let's order them to what a person is most likely interested in first.